src/Stripe.net/Entities/StripeInvoice.cs

```diff
@@ -92,6 +92,12 @@ internal object InternalCustomer
         [JsonProperty("forgiven")]
         public bool? Forgiven { get; set; }
 
+        [JsonProperty("hosted_invoice_url")]
+        public string HostedInvoiceUrl { get; set; }
+
+        [JsonProperty("invoice_pdf")]
+        public string InvoicePDF { get; set; }
```

Comment:
I think the convention would be `InvoicePdf` based off of uses of `Url` and `Id` elsewhere.